tokio-fs/src/file/os/unix/as_raw_fd.rs

```diff
@@ -0,0 +1,10 @@
+use std::os::unix::io::{AsRawFd, RawFd};
+
+use crate::File;
+
+impl AsRawFd for File {
+    fn as_raw_fd(&self) -> RawFd {
+        let std = self.std.clone();
```

Comment:
Why is the clone required?

Reply:
You're right. It doesn't need a clone() here. I must be thinking about something else.